src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -60,9 +60,13 @@ public ExploreService(
         this.parametersService = parametersService;
     }
 
-    public void createStudy(String studyName, UUID caseUuid, String description, String userId, UUID parentDirectoryUuid, Map<String, Object> importParams, Boolean duplicateCase) {
+    public record CaseInfo(UUID caseUuid, String caseFormat) {
```

Comment:
Why not creating a dedicated file for this DTO ? I think that's how we do in gridsuite.

Reply:
CaseInfo is not a DTO, i created it simply because sonar detected a code smell "to many parametters". so i fixed it by introducing a parameter object

Reply:
Same here
https://github.com/gridsuite/sensitivity-analysis-server/blob/9eed97f9daf845871e778973fa8cf165ba658465/src/main/java/org/gridsuite/sensitivityanalysis/server/dto/ReportInfos.java#L15
https://github.com/gridsuite/security-analysis-server/blob/865c75a020435b84cf5044bdd39bd63c757601c7/src/main/java/org/gridsuite/securityanalysis/server/dto/ReportInfos.java#L21

Reply:
done